applications/GeoMechanicsApplication/custom_constitutive/small_strain_udsm_2D_plane_strain_law.cpp

```diff
@@ -206,9 +203,10 @@ Vector& SmallStrainUDSM2DPlaneStrainLaw::
 }
 
 //----------------------------------------------------------------------------------------
-void SmallStrainUDSM2DPlaneStrainLaw::SetValue( const Variable<Vector>& rThisVariable,
-                                                const Vector& rValue,
-                                                const ProcessInfo& rCurrentProcessInfo )
+void SmallStrainUDSM2DPlaneStrainLaw::
+   SetValue( const Variable<Vector>& rThisVariable,
+             const Vector& rValue,
+             const ProcessInfo& rCurrentProcessInfo )
```

Comment:
```suggestion
void SmallStrainUDSM2DPlaneStrainLaw::SetValue(
    const Variable<Vector>& rThisVariable,
    const Vector& rValue,
    const ProcessInfo& rCurrentProcessInfo )
```
We normally use this style convention for methods with more than one argument.